src/error.rs

```diff
@@ -307,6 +311,9 @@ impl Error {
 impl Display for ErrorCode {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
+            ErrorCode::ControlCharacterWhileParsingString => {
```

Comment:
Keep this in the same order as in the `ErrorCode` enum.

Reply:
Done.